kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -207,6 +207,16 @@ export function ReportSettings(props: ReportSettingProps) {
       'write' | 'preview'
     >('write');
 
+    const handleHeader = (e) => {
+      setHeader(e);
+      reportDefinitionRequest.report_params.header = e;
+    };
```

Comment:
Any input validation for header and footer? Can user choose not to add any header and footer at all?

Reply:
Yes, header and footer can be blank- not sure what input validation to include, hasn't been discussed before.